rosserial_client/cmake/rosserial_client-extras.cmake

```diff
@@ -28,7 +28,6 @@ function(rosserial_generate_ros_lib)
     COMMAND ${CATKIN_ENV} rosrun ${make_libraries_PACKAGE} ${make_libraries_SCRIPT} ${PROJECT_BINARY_DIR}
   )
   add_custom_target(${PROJECT_NAME}_ros_lib DEPENDS ${PROJECT_BINARY_DIR}/ros_lib)
-  add_dependencies(${PROJECT_NAME}_ros_lib rosserial_msgs_genpy std_msgs_genpy)
```

Comment:
This probably shouldn't be removed. What's the rationale?

Reply:
This has been fixed a different way as of #320. Would you mind rebasing the rest of the change so that I can merge it?